src/components/dialogs/create-filter-dialog.js

```diff
@@ -270,9 +280,21 @@ const CreateFilterDialog = ({
                     row
                 >
                     <FormControlLabel
-                        value="FORM"
+                        value="Automatic"
+                        control={<Radio />}
+                        label={<FormattedMessage id="Automatic" />}
+                        defaultChecked={true}
```

Comment:
defaultChecked doesn't seems to be a correct prop for the Radio component nor the FormControlLabel component

Reply:
outdated comment it's ok now